src/styles/Blocks/Card.scss

```diff
@@ -0,0 +1,137 @@
+
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  border-radius: 5px;
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  font-family: "Roboto", Helvetica, sans-serif;
+  transition: transform 300ms, color 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__imagelement {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    padding-top: 32px;
+  }
+
+  &__description {
+    line-height: 18px;
+    display: flex;
+    flex-wrap: wrap;
+    width: 166px;
```

Comment:
You don't need to use fixed width for every block inside your card. Instaed of it you need to add paddings to your `card`